src/controller/place.last.selected.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { converterHospitalLastSelectedDTOToHospitalLastSelected } from "@/models/types/HospitalLastSelectedModel";
+import { StatusCode } from "@/models/types/StatusCode";
+import {
+  onCreatedLastSelectedPlace,
+  onGetLastSelectedPlace,
+} from "@/services/places.lastselected.service";
+import { Request, Response } from "express";
+
+/**
+ * Controlador para a rota /place/lastselected/create. Espera os seguintes parâmetros de consulta:
+ * @param userId: String
+ * @param hospitalName: String
+ * @returns HospitalLastSelected
+ * @throws NotFound
+ */
+async function placeLastSelectedCreateController(req: Request, res: Response) {
+  const userId = String(req.query.userId);
+  const hospitalName = String(req.query.hospitalName);
```

Comment:
Acho que nesse caso, essas informações poderiam estar no corpo da requisição (https://blog.rocketseat.com.br/tipos-de-parametros-nas-requisicoes-rest/). 

```suggestion
  const userId = String(req.body.userId);
  const hospitalName = String(req.body.hospitalName);
```